api/lib/infrastructure/repositories/user-repository.js

```diff
@@ -203,7 +203,9 @@ module.exports = {
 
   isEmailAvailable(email) {
     return BookshelfUser
-      .where({ email })
+      .query((qb) => {
+        qb.whereRaw('LOWER("email") LIKE ?', `${email.toLowerCase()}`);
```

Comment:
Les backticks n'ont pas d'utilité ici, non ?

```suggestion
        qb.whereRaw('LOWER("email") LIKE ?', email.toLowerCase());
```

Reply:
C'est la première fois que je vois un suggested change sur Github. Et c'est magnifique.

Reply:
C'est vraiment top comme feature, et ça fait gagner du temps je trouve 😄